.buildkite/steps/publish-image.sh

```diff
@@ -1,20 +1,21 @@
 #!/bin/bash
 set -euo pipefail
 
-IMAGETAG=${BUILDKITE_BRANCH:-master}
-BRANCHNAME=${BUILDKITE_BRANCH:-master}
+REVISION=$(git rev-parse HEAD)
+MASTER_REVISION=$(git rev-parse origin/master)
+DEVELOP_REVISION=$(git rev-parse origin/develop)
 
 docker images
 
 docker login -u=$DHUBU -p=$DHUBP
-docker push cyberway/cyberway.cdt:${IMAGETAG}
 
-if [[ "${IMAGETAG}" == "master" ]]; then
-    docker tag cyberway/cyberway.cdt:${IMAGETAG} cyberway/cyberway.cdt:stable
-    docker push cyberway/cyberway.cdt:stable
+if [[ "${REVISION}" == "${MASTER_REVISION}" ]]; then
+    TAG=stable
+elif [[ "${REVISION}" == "${DEVELOP_REVISION}" ]]; then
+    TAG=latest
```

Comment:
Why do you check here by revision? Please check by `BUILDKITE_BRANCH` as in cyberway.contracts https://github.com/cyberway/cyberway.contracts/blob/b84686a100c46194bf7f0c0246bfe8250ba4dd23/.buildkite/steps/publish-image.sh#L10-L16